helm/oauth2-proxy/README.md

```diff
@@ -196,6 +196,7 @@ The following table lists the configurable parameters of the oauth2-proxy chart
 | `service.loadBalancerIP`                              | ip of load balancer                                                                                                                                                                                                                                              | `nil`                                                                            |
 | `service.loadBalancerSourceRanges`                    | allowed source ranges in load balancer                                                                                                                                                                                                                           | `nil`                                                                            |
 | `service.nodePort`                                    | external port number for the service when service.type is `NodePort`                                                                                                                                                                                             | `nil`                                                                            |
+| `service.targetPort`                                  | name for the service's `targetPort`                                                                                                                                                                                                                              | `""`
```

Comment:
"A numeric port number (e.g., `80`) or a port name defined in the pod's container(s) (e.g., `http`)."

Reply:
changed in [a423d1d](https://github.com/oauth2-proxy/manifests/pull/270/commits/a423d1d151aa387935ee105b7845f81d47d9564f). Let me know if it's good to you